From d9ee24b30b748abb1cfc8387345342634cd6adf4 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:41 -0400
Subject: [redhat] kabi: Adding symbol unmap_mapping_range (mm/nommu.c)

Message-id: <acbfdb64336de8c03c313c01cb16fc1043bf19a9.1410510773.git.poros@redhat.com>
Patchwork-id: 96060
O-Subject: [PATCH RHEL7.1 221/233] kabi: Adding symbol unmap_mapping_range (mm/nommu.c)
Bugzilla: 1071099
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Unmap the portion of all mmaps in the specified
    address_space corresponding to the specified page range in the underlying file.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/unmap_mapping_range
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/unmap_mapping_range
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/unmap_mapping_range

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/unmap_mapping_range b/redhat/kabi/kabi-module/kabi_ppc64/unmap_mapping_range
new file mode 100644
index 0000000..c499b33
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/unmap_mapping_range
@@ -0,0 +1,2 @@
+#1-
+0x4da35905 unmap_mapping_range vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/unmap_mapping_range b/redhat/kabi/kabi-module/kabi_s390x/unmap_mapping_range
new file mode 100644
index 0000000..388c84c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/unmap_mapping_range
@@ -0,0 +1,2 @@
+#1-
+0x5af28e8d unmap_mapping_range vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/unmap_mapping_range b/redhat/kabi/kabi-module/kabi_x86_64/unmap_mapping_range
new file mode 100644
index 0000000..9b627d2
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/unmap_mapping_range
@@ -0,0 +1,2 @@
+#1-
+0x9373508b unmap_mapping_range vmlinux EXPORT_SYMBOL
-- 
1.7.1